pkg/discovery/contextresource.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 // GetContextResourcePath returns the file that will load the context resource from
-// here we only include files which have context information - no k8s or helm files
+// here. We only include files which have context information - no k8s or helm files
```

Comment:
Could someone double-check this?

Reply:
```suggestion
// GetContextResourcePath returns the file that will load the context resource.
// Here we only read files which have context information - no k8s or helm files
```

I think this would be more accurate. @jLopezbarb Can you confirm?

Reply:
Yes, I agree. That would be better